src/views/applicationManager.js

```diff
@@ -1,48 +1,138 @@
-const interview = {};
-
-// find page
-page = [];
-if (page == 0) {
-    page = 'login';
-} 
-else {
-//   destroy(lastPage);
-//    init(newPage);
-}
-
-//set up the collection of modules that app can use
+const navEvents = interview.addEventsNavigation();
```

Comment:
I'm gonna have to ask, just to make absolutely sure that the issue was properly understood: can you remind me what this is doing and why it was needed?